tests/conftest.py

```diff
@@ -952,28 +955,24 @@ def finalizer():
         """
         Delete the PVC
         """
-        pv_objs = []
 
-        # Get PV form PVC instances and delete PVCs
-        for instance in instances:
+        def operation_delete(instance):
             if not instance.is_deleted:
-                pv_objs.append(instance.backed_pv_obj)
+                pv_obj = instance.backed_pv_obj
                 instance.delete()
                 instance.ocp.wait_for_delete(instance.name)
+                if (
+                    pv_obj.data.get("spec").get("persistentVolumeReclaimPolicy")
+                    == constants.RECLAIM_POLICY_RETAIN
+                    and pv_obj is not None
```

Comment:
```suggestion
                    and pv_obj 
```

Reply:
To tell you the truth, I just want to take the code as is and first make it threaded. I have another PR which I did the same and also changed the original code and I got too much work in debugging it. I just want to parallel the existing code right now without any adventures.